### PR TITLE
My first whack at a P.R. I am likely to hose this up somehow, I guess.

### PR DESCRIPTION
**Pull request by whitedavidp**
1. Added write of line separator to modem.log within modem.py file.
2. Added /querynextcall to the webapp.

### CODE REVIEW

**Review by thess — CHANGES_REQUESTED** [review]
The original purpose of this logfile was to capture RAW output from the modem. The file is open in "binary" mode so things like line-endings, control characters, etc. are recorded. Adding arbitrary line-endings after the data subverts this process. In fact, I have been intending to remove this logging feature as it is no longer needed.

**Review by thess — CHANGES_REQUESTED** [review]
Please rebase, squash and force push the result and cleanup the comment. A single commit should suffice.

**Comment by thess** [comment]
I am not sure where to start here... There are several items here which need to be addressed. I hope you had a chance to at least review [CONTRIBUTING.md](https://github.com/thess/callattendant/blob/fmsentry/CONTRIBUTING.md) or [Github Creating a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) Please accept these comments as intended to help you create a good PR.

* First off the Subject/Title of this PR conveys no useful information about the proposed change - Please edit it to describe the fix/feature being submitted. Ex: "Add new API to Web interface to support feature X."

* The description of the PR does not explain what the feature is and/or how it is to be used. Submitting changes without a simple explanation of the problem being solved puts an expectation on the project developer to figure out what you are proposing. "This is good stuff, trust me..." is not useful.

* Commit messages need not feature a list of files being modified/updated unless being added for clarity. A simple explanation of the changes usually suffices.

* Ideally a PR should be a single commit and not the development *journey*. We would like to see the result or your work and not the process it took to get there. It also makes review easier. Please squash any fixes and/or additions and force push to your branch - this will automatically update the PR.

* **Important:** Please do not submit a PR from a branch that is under active development in the *upstream* repository. You should always create a local *patch* branch for your feature and submit from there. This avoids several complications that can arise moving forward. Most notably is when future main-line development introduces unintentional conflicts with your submission and it can no longer be merged. Or, worse, if the PR is rejected, you will then have to cleanup your fork or continuously "rebase" your changes. After a PR is accepted and merged, you can then delete your *patch* branch.

I hope these comments give you a better idea of what the submission process is about and the level of quality expected in the application. I suggest you close this PR and re-submit after incorporating this feedback.

Additional review comments are to be found in-line on your commits.

File: callattendant/hardware/modem.py
Comment (thess) [inline callattendant/hardware/modem.py:284]: The original purpose of this logfile was to capture RAW output from the modem. The file is open in "binary" mode so things like line-endings, control characters, etc. are recorded. Adding arbitrary line-endings after the data subverts this process. In fact, I have been intending to remove this logging feature as it is no longer needed.

File: callattendant/userinterface/webapp.py
Comment (thess) [inline callattendant/userinterface/webapp.py:860]: Please rebase, squash and force push the result and cleanup the comment. A single commit should suffice.

**Comment by whitedavidp** [comment]
All of this just seems like way too much effort for such small changes. And it is for reasons like my ignorance of the use/purpose of the modem.log that I asked about the value of some other changes from you and got rebuffed. Please do not think I do not appreciate all you have done and are doing. But I have no idea what "rebase, squash and force push the result" even means. So I have withdrawn this request as suggested. Sorry for bothering you.

**Pull request closed without merging**
